src/lexer_tok_utils.c

```diff
@@ -56,3 +56,19 @@ t_error	ft_add_to_buf(char c, t_buf *tmp)
 	}
 	return (SUCCESS);
 }
+
+t_error	ft_add_double_redirect(t_src *src, char *c, t_buf *tmp)
+{
+	char	next;
+	t_error	err;
+
+	err = ft_peek_char(src, &next);
+	if (err != SUCCESS)
```

Comment:
Just for clarification: This would only fire if we are - why ever - on position 0 in the src string right? So it's like checking for empty vars although we always pass vars